lib/transformations.js

```diff
@@ -1,19 +1,18 @@
 'use strict';
 
-const row = (
+const row =
   // Get dataset row
-  ds // array of records, example: [ { Id: 1, Name: 'Marcus' } ]
+  ds => { // array of records, example: [ { Id: 1, Name: 'Marcus' } ]
   // Result: result array of records, example: [1, 'Marcus']
-) => {
```

Comment:
This breaks function signature parser, so if we will remove round brackets for functions with single argument, we should change style of comments too @belochub